lib/components/app/app.css

```diff
@@ -47,36 +47,266 @@
   }
 }
 
+/* Navbar settings */
+
+.navbar .navbar-header {
```

Comment:
One of these settings alters the vertical positioning of the agency logo in desktop wide view (See screnshot), that means we need CSS to fix that.

<img width="312" alt="image" src="https://github.com/user-attachments/assets/5594597b-c7d2-454c-bd73-8808efa178b1">

Reply:
A lot of the existing logo positioning css you're seeing here is coming from the branding logo css in the configuration. This PR should ideally remove the need for that, so I've opened an internal PR to adjust these margins and tagged you there as well!